rsocket-core/src/main/java/io/rsocket/lease/LeaseSupport.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.rsocket.lease;
+
+import io.netty.buffer.ByteBufAllocator;
+import java.util.function.BiConsumer;
+
+public class LeaseSupport {
```

Comment:
Exists to reduce number of null checks in RSocketFactory and RSocketRequester/Responder